Packs/CommonScripts/Scripts/ReadPDFFileV2/ReadPDFFileV2.py

```diff
@@ -494,7 +494,7 @@ def main():
                 # Get metadata:
                 metadata = get_pdf_metadata(cpy_file_path, user_password)
 
-                if user_password:  # The PDF is encrypted
+                if user_password or 'yes' in metadata.get('Encrypted'):  # The PDF is encrypted or has AES encryption
```

Comment:
Does this decrypt the pdf?

Reply:
Yes. The pikepdf package knows how to handle this encryption.